### DISCUSSION TITLE
docsify search plugin excluding certain page

### DISCUSSION THREAD

**Discussion by michaelmartinth** [docsifyjs/docsify#2161] [Q&A]
Hi, I am wondering, is it possible to exclude certain pages from search results?
For example, I have changelog in my docsify, which is a requirement but not relevant for most reader.
Is it possible to 'blacklist' the changelog folder from search plugin?

**Comment by Koooooo-7** [comment]
Currently, it is no one step option. 
As a workaround,  I think you could register a hook and manually check and remove those unnecessary contents in the search index (localStorage).
